src/main/java/com/minecolonies/rankup/util/AccountingUtils.java

```diff
@@ -0,0 +1,568 @@
+        }
+        return "";
+    }
+
+    public int getPlayerTime(final UUID uuid)
+    {
+        try
+        {
+            if (getPlugin().getModuleContainer().isModuleLoaded(DatabaseModule.ID))
+            {
+                final ResultSet results = getQuery(SELECT + " " + TIME_PLAYED_COLUMN + " " + FROM + " " + tableId + " " + WHERE + " " + UUID_COLUMN + " = '" + uuid + "'");
+
+                if (results != null && results.next())
+                {
+                    return results.getInt(TIME_PLAYED_COLUMN);
+                }
+            }
+            else
+            {
+                return getAccountConfig().playerData.get(uuid).timePlayed;
+            }
+        }
+        catch (SQLException | NoModuleException e)
+        {
+            getPlugin().getLogger().warn("Get Player Time failed", e);
+        }
+        return 0;
+    }
+
+    private void updatePlayerAttribute(final UUID uuid, final String column, final String attribute, final boolean isDate) throws SQLException
+    {
+        if (getConn() != null)
+        {
+            String statement = UPDATE + " "
+                                 + tableId + " "
+                                 + SET + " "
+                                 + getColumn(column) + " = "
+                                 + "? "
+                                 + WHERE + " "
+                                 + UUID_COLUMN + " = "
+                                 + "?";
+
+            try (PreparedStatement stmt = getConn().prepareStatement(statement))
+            {
+
+                if (isDate)
+                {
+                    stmt.setDate(1, Date.valueOf(CommonUtils.dateFormat(attribute)));
+                    stmt.setString(2, uuid.toString());
+                    stmt.execute();
+                }
+                else
+                {
+                    stmt.setString(1, attribute);
+                    stmt.setString(2, uuid.toString());
+                    stmt.execute();
+                }
+            }
+        }
+    }
+
+    private void updatePlayerAttribute(final UUID uuid, final String column, final int attribute) throws SQLException
+    {
+        if (getConn() != null)
+        {
+
+            String statement = UPDATE + " "
+                                 + tableId + " "
+                                 + SET + " "
+                                 + getColumn(column) + " = "
+                                 + "? "
+                                 + WHERE + " "
+                                 + UUID_COLUMN + " = "
+                                 + "?";
+
+            try (PreparedStatement stmt = getConn().prepareStatement(statement))
+            {
+                stmt.setInt(1, attribute);
+                stmt.setString(2, uuid.toString());
+                stmt.execute(); // was here, ''TIME_PLAYED' = 0 WHERE UUID = '1af3cf96-5018-4fed-a96f-35eddbe402d7''
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)